### PR TITLE
added models.py, moved dynamodb methods from statistics.py to dynamod…

### PR DESCRIPTION
**Pull request by MagdalenaBialik**
…b_dao.py

### CODE REVIEW

File: app/statistic_class.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    s3_bucket_name: str
+    days: Optional[int]
+    email_title: str
+
+
+class Statistic:
+    def __init__(
+        self, dynamodb_table, s3_bucket, ses_service, settings: StatisticsSettings
+    ):
+        self.dynamodb_table = dynamodb_table
```
Comment (FidelusAleksander) [inline app/statistic_class.py:19]: create dynamodb_dao here

File: app/statistic_class.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        self.s3_bucket = s3_bucket
+        self.ses_service = ses_service
+        self.settings = settings
+
+    def get_object_from_s3(self, pet_statistics: List[PetStatistics]):
```
Comment (FidelusAleksander) [inline app/statistic_class.py:24]: get_presigned_url

File: handlers/mail_statistics.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+from app.statistic_class import Statistic, StatisticsSettings
+
+settings = StatisticsSettings()
+
+db_table = boto3.resource(service_name="dynamodb", region_name="eu-west-1").Table(
+    settings.dynamodb_table_name
+)
+
+s3 = boto3.client(service_name="s3", region_name="eu-west-1")
```
Comment (FidelusAleksander) [inline handlers/mail_statistics.py:13]: here or in next PR use s3 bucket instead of client

Comment (FidelusAleksander) [reply handlers/mail_statistics.py:13]: this will have implications on Statistic class

File: app/statistic_class.py [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    days: Optional[int]
+    email_title: str
+
+
+class Statistic:
+    def __init__(
+        self, dynamodb_table, s3_bucket, ses_service, settings: StatisticsSettings
+    ):
+        self.dynamodb_table = dynamodb_table
+        self.s3_bucket = s3_bucket
```
Comment (FidelusAleksander) [inline app/statistic_class.py:20]: currently you are passing s3_client, either pass s3_bucket or rename to s3_client

**Review by FidelusAleksander — CHANGES_REQUESTED** [review]

File: app/statistic_class.py (hunk 2 above)
Comment (MagdalenaBialik) [reply app/statistic_class.py:24]: ok

File: app/statistic_class.py [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+        s3_client,
+        ses_service,
+        settings: StatisticsSettings,
+        dynamodb_dao: DynamoDBDao,
+    ):
+        self.dynamodb_table = dynamodb_table
+        self.s3_client = s3_client
+        self.ses_service = ses_service
+        self.settings = settings
+        self.dynamodb_dao = dynamodb_dao
```
Comment (FidelusAleksander) [inline app/statistic_class.py:28]: self.dynamodb_dao = DynamoDBDao(dynamodb_table=dynamodb_table, settings=settings)

File: app/statistic_class.py (hunk 1 above)
Comment (FidelusAleksander) [reply app/statistic_class.py:24]: remove the self.dynamodb_table attribute

File: app/statistic_class.py [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+
+class Statistic:
+    def __init__(
+        self,
+        dynamodb_table,
+        s3_client,
+        ses_service,
+        settings: StatisticsSettings,
+        dynamodb_dao: DynamoDBDao,
```
Comment (FidelusAleksander) [inline app/statistic_class.py:22]: remove this parameter, pass dynamodb_table to statistics and create DynamoDB dao from it

**Review by FidelusAleksander — CHANGES_REQUESTED** [review]

**Review by FidelusAleksander — APPROVED** [review]

**Pull request merged**